microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md

```diff
@@ -73,6 +73,23 @@ Issues are evaluated live while you edit your code.  As you type, you get feedba
 
 ![A possible issue being explained on an output element](microsoft-edge-devtools-extension-images/live-issue-reporting.png)
 
+#### Automated fixes and issue filtering
+
+When you hover over the element with the issue, you get a lightbulb icon indicating that there are quick fixes available.
+
+![An anchor element with a protocol specific href attribute highliighted as a problem with a lightbulb icon above it](microsoft-edge-devtools-extension-images/light-bulb.png)
+
+Activating the lightbulb icon shows a list of options. For example, if you added a link with a protocol relative URL, you get:
+
+![The quick fix panel opened next to the anchor element with the error offering the different options explained in the following text](microsoft-edge-devtools-extension-images/quick-fix-options.png)
```

Comment:
This 2nd .png shows "example.com", might be a requirement to change it to "contoso.com" and change demo code to that.

Reply:
I don't have a button to Resolve conversation, and I don't want to Delete this comment which was substantive.
@Reezaali can you Resolve the present conversation?